barretenberg/cpp/src/barretenberg/plonk/proof_system/widgets/transition_widgets/arithmetic_widget.hpp

```diff
@@ -103,32 +73,13 @@ template <class Field, class Getters, typename PolyContainer> class ArithmeticKe
         const Field& q_c =
             Getters::template get_value<EvaluationType::NON_SHIFTED, PolynomialIndex::Q_C>(polynomials, i);
 
-        Field result = linear_terms[0] * q_m;
-        result += (linear_terms[1] * q_1);
-        result += (linear_terms[2] * q_2);
-        result += (linear_terms[3] * q_3);
+        Field result = (w_1 * w_2) * q_m;
+        result += w_1 * q_1;
+        result += w_2 * q_2;
+        result += w_3 * q_3;
         result += q_c;
         result *= alpha;
-        return result;
-    }
-
-    /**
-     * @brief Compute the scaled values of openings
-     *
-     * @param linear_terms The original computed linear terms of the product and wires
-     * @param scalars A map where we put the values
-     * @param challenges Challenges where we get the alpha
-     */
-    inline static void update_kate_opening_scalars(coefficient_array& linear_terms,
```

Comment:
This function (implemented in each widget) was part of the linearization trick and was no longer used at all.